internal/guest/runtime/runc/runc.go

```diff
@@ -139,7 +144,7 @@ func (c *container) Start() error {
 	if err != nil {
 		runcErr := getRuncLogError(logPath)
 		c.r.cleanupContainer(c.id)
-		return errors.Wrapf(err, "runc start failed with %v: %s", runcErr, string(out))
+		return errors.Wrapf(runcErr, "runc start failed with %v: %s", err, string(out))
```

Comment:
why did we swap these?

Reply:
since we are are converting the error from the log file to a proper `error`, having it as the first argument allows for using `errors.Is`.

Reply:
Okay that's fine